src/start_menu/main_settings/speed.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+#include "../start_menu.h"
+
+void Speed();
```

Comment:
Provide an empty line at the end of the .cpp and .h files.

Also, more descriptive names and comments above the declarations (see weight.h for similar comment).

Reply:
done